### PR TITLE
fix: allow user to set any day as starting day of week

### LINKED ISSUE #761: Allow any weekday as first day of calendar
#### Is this a bug, enhancement, or feature request?
Bug

For some reason we can only set the start day to be Sunday or Monday.

### PR DESCRIPTION
**Pull request by mikerodonnell89**
…datepicker and datetimepicker.  Also fixes bug where days of the week did not show up in the date selection grid.

fixes #761

### CODE REVIEW

File: library/src/lib/datetime-picker/datetime-picker.component.ts [hunk 1]
```diff
@@ -61,6 +61,9 @@ export class DatetimePickerComponent implements OnInit, ControlValueAccessor {
     @ViewChild(TimeComponent)
     timeComponent;
 
+    @Input()
+    startingDayOfWeek: number = 0;
```
Comment (MattL75) [inline library/src/lib/datetime-picker/datetime-picker.component.ts]: Need to ensure this is between 0 and 6, no?

Comment (mikerodonnell89) [reply library/src/lib/datetime-picker/datetime-picker.component.ts]: There's a note in the docs about 1 being monday, 2 being tuesday etc.  But will limit this to 0-6

Comment (MattL75) [reply library/src/lib/datetime-picker/datetime-picker.component.ts]: ideally they should match internally & externally

Comment (mikerodonnell89) [reply library/src/lib/datetime-picker/datetime-picker.component.ts]: Can you elaborate, not sure what you mean by this

Comment (MattL75) [reply library/src/lib/datetime-picker/datetime-picker.component.ts]: like the value being input to the component should be the same as the one being used internally

**Comment by MattL75** [comment]
image
image

Days are different. In the calendar below April 30th does not appear.

**Pull request merged**